### DISCUSSION TITLE
Import / Export option for TaxoPress features

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-taxonomies#1549] [Feature Requests]
> I need a tool to export and import the TaxoPress configuration? Because I'm on multisite and taxopress is active on every site and I would have to replicate the configuration for 23 sites.

**Comment by ElFreshPrince** [comment]
I spent an hour today trying to find this function i thought i was just missing it, crazy it doesent exist with so many custom settings
